docs/New-Maintainer-Checklist.md

```diff
@@ -50,15 +50,27 @@ Thanks for all your work so far!
 
 If they accept, follow a few steps to get them set up:
 
-- Invite them to the [**@Homebrew/maintainers** team](https://github.com/orgs/Homebrew/teams/maintainers) to give them write access to all repositories (but don't make them owners yet). They will need to enable [GitHub's Two Factor Authentication](https://help.github.com/articles/about-two-factor-authentication/).
+- Invite them to the [**@Homebrew/maintainers** team](https://github.com/orgs/Homebrew/teams/maintainers) (or any relevant [subteams](https://github.com/orgs/Homebrew/teams/maintainers/teams)) to give them write access to relevant repositories (but don't make them owners). They will need to enable [GitHub's Two Factor Authentication](https://help.github.com/articles/about-two-factor-authentication/).
 - Ask them to sign in to [Bintray](https://bintray.com) using their GitHub account and they should auto-sync to [Bintray's Homebrew organisation](https://bintray.com/homebrew/organization/edit/members) as a member so they can publish new bottles
 - Add them to the [Jenkins' GitHub Authorization Settings admin user names](https://jenkins.brew.sh/configureSecurity/) so they can adjust settings and restart jobs
 - Add them to the [Jenkins' GitHub Pull Request Builder admin list](https://jenkins.brew.sh/configure) to enable `@BrewTestBot test this please` for them
 - Invite them to the [`homebrew-maintainers` private maintainers mailing list](https://lists.sfconservancy.org/mailman/admin/homebrew-maintainers/members/add)
 - Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites) (and ensure they've read the [communication guidelines](Maintainer-Guidelines.md#communication))
-- Invite them to the [`homebrew` private maintainers 1Password](https://homebrew.1password.com/signin)
+- Add them to [Homebrew's README](https://github.com/Homebrew/brew/edit/master/README.md)
+
+If they are also interested in doing system administration work:
+
+- Invite them to the [`homebrew-ops` private operations mailing list](https://lists.sfconservancy.org/mailman/admin/homebrew-ops/members/add)
+- Invite them to the [`homebrew` private 1Password](https://my.1password.com/people)
```

Comment:
Don’t link to `my.1password.com`. Whoever follows that link is going to end up in the admin area of the account they have most recently used. A lot of 1Password users are members of different accounts (e. g., a Teams account at work), and chances are they use the latter much more often.

This is not only a UX but also a security issue. If an admin is stressed or in a hurry, and trusts the link, the admin may grant an unsuspecting Homebrew maintainer access to company secrets in the worst case.

This can be avoided by linking to the account directly, e. g. `https://homebrew.1password.com/people`.

Reply:
Good idea, fixed, thanks @claui.